AopCaching.NuGet/AopCaching.NuGet.csproj

```diff
@@ -40,7 +40,6 @@
     <SkipPostSharp>True</SkipPostSharp>

```

Comment:
This file can be deleted, as well as the entire proj. Same goes for all *.NuGet projects.